dali/test/python/test_dali_cpu_only.py

```diff
@@ -46,7 +46,9 @@
 coco_dir = os.path.join(data_root, 'db', 'coco', 'images')
 coco_annotation = os.path.join(data_root, 'db', 'coco', 'instances.json')
 sequence_dir = os.path.join(data_root, 'db', 'sequence', 'frames')
-video_files = get_files(os.path.join('db', 'video', 'vfr'), 'mp4')
+video_files = [
+    os.path.join(get_dali_extra_path(), 'db', 'video', 'vfr', 'test_1.mp4'),
+    os.path.join(get_dali_extra_path(), 'db', 'video', 'vfr', 'test_2.mp4')]
```

Comment:
Before we were listing all files in the directory. We are adding new files to this directory in other PRs. This caused this test case to get bigger due to some unrelated changes. I decided to change this behavior.